### PR TITLE
Tensorflow warning on graph optimization

### PR DESCRIPTION
**Pull request by Simosound94**
**ISSUE**
If the package is used without installation, tensorflow produces a warning during the graph optimization procedure (https://github.com/tensorflow/tensorflow/blob/1ae01a07f5e4b2911218d5fc3419b3eaed48b75b/tensorflow/core/grappler/optimizers/arithmetic_optimizer.cc#L3152).
Warnings are suppressed if the repo is used through the python packaging, however the problem remains unsolved.

The warnings are produced through
`boxes, scores, labels = model.predict_on_batch(np.expand_dims(image, axis=0))`

**BUG REPORT**
- tensorflow==1.13.1
- Keras==2.3.0

```
git clone https://github.com/fizyr/keras-retinanet.git
cd keras-retinanet
python setup.py build_ext --inplace
cd examples
python ResNet50RetinaNet.py

2019-09-27 10:42:54.842469: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node clipped_boxes/strided_slice_2. Error: Pack node (boxes/stack) axis attribute is out of bounds: 2
2019-09-27 10:42:54.842516: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node clipped_boxes/strided_slice_3. Error: Pack node (boxes/stack) axis attribute is out of bounds: 2
2019-09-27 10:42:54.842532: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node clipped_boxes/strided_slice_4. Error: Pack node (boxes/stack) axis attribute is out of bounds: 2
2019-09-27 10:42:54.842545: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node clipped_boxes/strided_slice_5. Error: Pack node (boxes/stack) axis attribute is out of bounds: 2
2019-09-27 10:42:55.355164: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node clipped_boxes/strided_slice_2. Error: Pack node (boxes/stack) axis attribute is out of bounds: 2
2019-09-27 10:42:55.355228: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node clipped_boxes/strided_slice_3. Error: Pack node (boxes/stack) axis attribute is out of bounds: 2
2019-09-27 10:42:55.355269: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node clipped_boxes/strided_slice_4. Error: Pack node (boxes/stack) axis attribute is out of bounds: 2
2019-09-27 10:42:55.355290: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node clipped_boxes/strided_slice_5. Error: Pack node (boxes/stack) axis attribute is out of bounds: 2
```

**PROPOSAL**
- the code of examples/ResNet50RetinaNet.py has been added to reproduce the issue (it's the same of examples/ResNet50RetinaNet.ipynb. but keras_retinanet is imported from folder)
- the custom keras layer ClipBoxes now supports tensorflow optimization

### CODE REVIEW

**Comment by hgaiser** [comment]
Ah nice, thank you for looking into this! I've seen the warning before but never looked into the cause.
